src/components/movies/comments.jsx

```diff
@@ -32,8 +32,8 @@ export const Comments = () => {
     useSocket({
         type: "NEW_COMMENT",
         callBack: (payload) => {
+            console.log(payload.data);
```

Comment:
Ne znam ni do sada sto sam dopustao console.log da ide, ali svaki code reviewer ikada ce smarati za ovo da nikako ne ide u codebase 🙂
Okej u onim situacijama kada catch-ujes error, to se i inace tako radi da se console log-uje, ali ne zelimo.
Kontam da ti je jasno to i bez ove price, ali samo zelim da imas u vidu da se to strogo ne dozvoljava na komercijalnim projektima 😄

Reply:
Obrisao sam sve osim kod hvatanja gresaka.